### ISSUE TITLE
Cannot see the scratches option under project. Help needed

### ISSUE THREAD

**Issue by chernandezosma** [ChrisRM/material-theme-jetbrains#614]
### Summary

I can't see the scratches option under project in side panel.

### Steps to reproduce (if needed)

1. Open a project
2. Click on Project up of tree view 
3. I cannot see the scratches option

You can also add Screenshots for a better understanding.

![screen shot 2018-01-26 at 10 26 04](https://user-images.githubusercontent.com/7124175/35435635-6413eaba-0283-11e8-9e6d-7cadffa70d57.png)

### Environment

=====PERFORMANCE SUMMARY=====
Filesystem Info:
File system is case sensitive: false
Are there symlinks: true
Number of files: 62811

System Info:
Number of CPU: 8
Used memory: 175Mb 
Free memory: 84Mb 
Total memory: 260Mb 
Maximum available memory: 1981Mb 
Displays: 
Display 0: 3360x2100; scale: 2.00
Display 1: 1920x1080; scale: 1.00
Display 2: 1920x1080; scale: 1.00

Project Info:
Number of opened files: 6
File size (in lines): 2357
File size in characters: 88379
Last action: _Undo
Number of injection: 0

IDE Info:
Custom plugins: [Color Ide (1.3), Dummy Text Generator (1.0.11), Quick Notes (2.9.6), Dash (3.3), Afterglow Theme (1.0.8), TypoScript Plugin (1.9.0), .ignore (2.4.0), BashSupport (1.6.12.173), Laravel Plugin (0.15), PHP Annotations (5.2.1), Material Theme UI (1.6.0.4), Symfony Plugin (0.16.163.1), GitToolBox (173.2.1)]
Disabled plugins:[]
Build version: PhpStorm 2017.3.3 Build #PS-173.4301.34 January 18, 2018
Java version: 1.8.0_152-release-1024-b11x86_64
Operating System: Mac OS X (10.13.2, x86_64)
JVM version: OpenJDK 64-Bit Server VM JetBrains s.r.o

**Comment by mallowigi** [comment]
Have you tried with the plugin off?

**Comment by chernandezosma** [comment]
@mallowigi Yes, if I disable the plugin the option works perfectly, in another instance of PHPStorm with the plugin the option is enabled and visible, but not in this instance. I no have any clue about what is happening.

All instances were installed in the same way.

**Comment by mallowigi** [comment]
I will take a look but in my opinion they moved it elsewhere.
